### ISSUE TITLE
Not accepting privacy policy prevents "Verification email sent" display

### ISSUE THREAD

**Issue by dougbu** [aspnet/Templating#791]
When using Bootstrap 4 / .NET Core 2.2 Identity (Individual Auth) templates, "Send verification email" link does not display "Verification email sent" box unless Privacy has been accepted.
#### Repro steps
1. Open Visual Studio after installing latest .NET Core SDK
2. File --> New --> Project&hellip;
3. Click on ASP.NET Core Web Application
4. Click on Web Application or Web Application (Model-View-Controller)
5. Confirm ASP.NET Core 2.2 is selected in rightmost drop-down control
6. Click Change Authentication
7. Click Individual User Accounts
8. Click :ok:
9. Click :ok:
10. Debug --> Start Without Debugging
11. If necessary, delete cookies for localhost and refresh home page to display the Privacy / Content acceptance banner
12. Click Register
13. Fill in form and click :ok:
14. Click Apply Migrations
15. Refresh page
16. Click Hello {your email}
17. Click Send verification email

##### Expected
![image](https://user-images.githubusercontent.com/6431421/47107555-7fc02f80-d1fe-11e8-9619-80ada35c39dd.png)
##### Actual
![image](https://user-images.githubusercontent.com/6431421/47107627-a67e6600-d1fe-11e8-87f0-9656d69182d4.png)

Found while verifying aspnet/Release#324

**Comment by mkArtakMSFT** [comment]
@javiercn, can you please look into this. We should fix this as part of 2.2 release.

**Comment by javiercn** [comment]
@mkArtakMSFT This is by design. What Doug is referring to is the cookie policy, which governs non-essential cookies, like the temp data one. This was discussed at the time and we agreed that it is not guaranteed for the site to work if the cookie policy is not accepted. 

This can be opted out by making the temp data cookie essential.
```csharp
services.AddMvc()
   .AddCookieTempDataProvider(o => o.Cookie.IsEssential = true);
```
